README.md

```diff
@@ -270,28 +290,32 @@ Let's put `node-1` into maintenance mode:
 $ swarmctl node drain node-1
 
 $ swarmctl node ls
-ID             Name    Membership  Status  Availability  Manager status
---             ----    ----------  ------  ------------  --------------
-2o8evbttw2sjj  node-1  ACCEPTED    READY   DRAIN         REACHABLE
-2p7w0q83jargg  node-2  ACCEPTED    READY   ACTIVE        REACHABLE *
-3ieflj99g4wh8  node-3  ACCEPTED    READY   ACTIVE        REACHABLE
+ID                         Name    Membership  Status  Availability  Manager Status
+--                         ----    ----------  ------  ------------  --------------
+7cna6u5nqcmi7moe9qnjzrss5  node-1  ACCEPTED    READY   DRAIN         REACHABLE *
+cr962qysr3xr6k3ed5a5qjsde  node-3  ACCEPTED    READY   ACTIVE
+du359trx1fnyhztwwpjlpa9yl  node-2  ACCEPTED    READY   ACTIVE
 
 $ swarmctl service inspect redis
-ID                : 89831rq7oplzp6oqcqoswquf2
-Name              : redis
-Replicas          : 6
+ID                   : dy0vwwae8ropb5skvzcekzk6o
+Name                 : redis
+Replicas             : 6/6
+Update Status
+ State               : PAUSED
+ Started             : 1 minute ago
+ Message             : update paused due to failure or early termination of task 77k803vw9jvzk3z6hhwlk0rni
```

Comment:
Something went wrong with your upgrade, so the output below shows a mix of redis:3.0.6 and redis:3.0.7.

I think you need to wait for the rolling update to finish before draining node-1.